### ISSUE TITLE
Is there any demo code about how to use apex for expectation maximization?

### ISSUE THREAD

**Issue by YunseokJANG** [NVIDIA/apex#745]
I've checked https://github.com/NVIDIA/apex/issues/374 this issue recently, but I have an additional question when I want to apply EM (=> there should be a set of partial updates, but the distributed training requires to have gradient signal from all modules). 

In such a scenario, I may able to feed the loss for the E step, and accumulate the loss for the M step, and then update at once. However, if I want to update the E part first and then M part, how I can implement such schedule in the APEX/distributed training setting?
